src/viadot/sources/vid_club.py

```diff
@@ -0,0 +1,400 @@
+"""Vid Club Cloud API connector."""
+
+from datetime import datetime, timedelta
+from typing import Any, Dict, List, Literal, Tuple
+import pandas as pd
+
+from prefect import get_run_logger
```

Comment:
Remove this pls, sources must not refer to Prefect it's the other way around, orchestration is a layer on top of viadot sources.

Reply:
I do not see it anymore. Probably Fabio changed that already

Reply:
Now I see this https://github.com/dyvenia/viadot/pull/1044/files/e54e0040c2e07b29e055982ee9ea41f433c4bbed..b5c0405ad399f03768630c2926128a6e228bcff2#diff-b689e672f459fbee00840eb09808eaaa447f1f234f47c613ea43e8ab6d936721R14-R16, which is still incorrect. `self.logger` should simply be used (it's defined in the `Source` class from which this connector is inheriting).

Reply:
@Rafalz13 still there